reddash/app/home/routes.py

```diff
@@ -547,6 +547,33 @@ def fetchaliases(guild):
 # ----------------------------------- Third Party -----------------------------------
 
 
+@blueprint.route("/api/webhook", methods=("POST",))
+def webhook_route():
+    # Webhook received
+    if not request.is_json:
+        # reject any requests that aren't json for now
+        return jsonify({"status": 0, "message": "Not connected to bot"})

```

Comment:
Catching this before you go through hell of resolving conflicts.  A message like `Invalid formatting` may be better in this situation, as we usually output `Not connected to bot` when there was an RPC issue.